src/signals/incident/components/form/Error/Error.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Header from '../Header';
+import { InvalidatedProjectKind } from 'typescript';
+
+const Error = ({ meta, parent: { touched, valid } }) => {
+  const hasError = errorType => errorType === 'global' && !valid;
+  const getError = errorType => (errorType === 'global' && meta.label) || true;
+  return <Header meta={{ name: 'global' }} touched={touched} hasError={hasError} getError={getError} />;
```

Comment:
Do we really need the Header component here? How about simply returning a <Paragraph /> with text "Er zijn vragen niet (of niet juist) ingevuld. Vul de vragen hieronder op de goede manier in."? And perhaps a rename of the component to GlobalError?

Reply:
Makes sense actually. Since not much of the `Header` is being used anymore and even exceptions have been added.